modules/matlab/include/opencv2/matlab/bridge.hpp

```diff
@@ -522,6 +522,11 @@ class Bridge {
   Bridge& operator=(const Ptr_TonemapReinhard& ) { return *this; }
   Ptr_TonemapReinhard toPtrTonemapReinhard() { return Ptr_TonemapReinhard(); }
   operator Ptr_TonemapReinhard() { return toPtrTonemapReinhard(); }
+
+  // ---------------------------   Ptr_float   ----------------------
+  Bridge& operator=(const Ptr_float& ) { return *this; }
+  Ptr_float toPtrFloat() { return Ptr_float(); }
```

Comment:
This is just to fix the build process. If they want to make sure the code is doing something useful, than that is for the guy of the other PR that wrapped this wrongfully xD